Tasks/XamariniOS/task.json

```diff
@@ -112,7 +112,7 @@
             "required": false,
             "groupName": "sign",
             "defaultValue": "file",
-            "helpMarkDown": "Indicates that the build should use different signing and/or mobile provisioning settings than are installed on the build server. Leave fields blank if you do not wish to override.",
+            "helpMarkDown": "If the build should use a different signing or provisioning method than is installed on the build agent, choose that method here. Choose 'File Contents' to use a P12 certificate and provisioning profile. Choose 'Identifiers' to retrieve pre-installed signing settings from the Keychain. Leave the corresponding fields blank if you do not wish to override default settings on the build agent.",
```

Comment:
When choosing 'Identifiers', I would say, "to retrieve pre-installing signing settings from the default keychain and profiles" since profiles are stored outside the keychain as files on disk.